pkg/webhook/vm_validatation.go

```diff
@@ -108,3 +125,80 @@ func (vmValidator *vmDeviceHostValidator) validateDevicesFromSameNodes(vmObj *ku
 
 	return nil
 }
+
+func (vmValidator *vmDeviceHostValidator) validateHostDevices(vmObj *kubevirtv1.VirtualMachine) error {
+	for _, hostDevice := range vmObj.Spec.Template.Spec.Domain.Devices.HostDevices {
```

Comment:
The object access path `vmObj.Spec.Template.Spec.Domain.Devices.HostDevices` is quite long, maybe we can consider extracting it as a utility function.

Reply:
We can extract it later if we truly utilize a significant number of these paths.